### DISCUSSION TITLE
機械系對比電機系

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#985] [Q&A]
純粹討論，為什麼機械系在台灣被打壓的這麼嚴重，明明科系跟電機差不多難讀，但是為什麼顛機系出路就比較好，不能因為護國神山的關係就讓一大堆人無腦選擇電機系吧，這樣台灣社會以後這座神山倒了，不就沒有任何競爭能力了嗎?
為什麼現在的策略不是讓台灣多方涉略，而是專注一項專長?不太能理解

**Comment by ruthchen1020** [comment]
因為現在設計IC薪水最高，沒有設計機械的需求，只有設備很缺啦

**Comment by 12Jesse24** [comment]
確實大家都偏向選高薪的，就像同樣工作8小時你要拿1000元還是10元，雖然比喻誇張了點就是了==
但還是推薦選興趣，比如說寫程式就是我的全部，可以dbug10小時不睡覺還是很爽，抑或是操作機台好開心壓那個會這樣動怎麼那麼酷，可以壓個沒日沒夜這種，所以還是建議你在大學期間多去尋找自己的興趣，加油啦!

**Comment by ATREE01** [comment]
哪邊錢多人就往哪邊走阿，而且台灣因為只有這類ICT產品出口到其他國家免稅，能夠賺取比較高的利潤，因此整個國家產業發展就往這邊片移了。

**Comment by Chengbosco** [comment]
也有可能機械系有種髒髒的形象(不知道不要炮我)?加上錢也不多，所以當然沒人想讀，但這對台灣的未來當然不太好啦。

**Comment by Yanxxx7979** [comment]
聽過學長分析，由於台灣的工業鏈從重工業跳過精密機械產業，直接進入半導體業，而精密機械產業正是機械系最直接就業選擇，進而導致機械系出路較為慘淡

**Comment by uwje** [comment]
機械系的出路感覺比較廣，但也比電機來的窄，現在的機械系已經不太算黑手，反而轉往多功能的科系發展，如半導體製程等先進產業

**Comment by spermwhale464** [comment]
電機系畢業後就是機械系的上游，他們設計完的晶片丟給機械系做，一個被動一個主動，再加上台灣這種奴性文化，所以機械系的出路就沒有比較被看好

**Comment by Tomken123** [comment]
老子不想讀書了，想休學出社會，管它這麼多，我有錢拿就好了，過的爽就好

**Comment by KH1126** [comment]
我覺得電機系出路較廣，機械系學的多，但不精。
像是電子電路學學一學期，電機系電子學可以開到電子三。

**Comment by William08290829** [comment]
我覺得真的就是長期下來台灣產業界的問題，像我之前也是讀機械系的，也知道沒有所謂的哪系好讀哪系不好讀，要怪感覺真的只能怪這社會就是給電機比較多錢。或許等哪天電機的人才飽和了就可能可以緩解這現象吧。

**Comment by EPHMR0524** [comment]
硬要說的話就還是產業結構的問題吧,大部分高薪的公司都是做半導體的，而剛好電機系正是在這方面的相關課程比較多，所以也因此比較多人投入，但老實講在和機械系差不多累的情況，薪水硬是比機械系畢業的高出一截，老實講挺不公平的吧，但有時選擇就是比較重要。

**Comment by jeff110303549** [comment]
機械工程和電機工程都是非常重要的學科，兩者在科學和技術上都扮演著關鍵的角色。雖然機械工程在台灣可能受到一些限制，但這不應該影響學生對自己興趣和專業的選擇。

電機工程可能在某些方面受到更多重視，尤其在現在這個半導體產業蓬勃發展的趨勢下，然而這並不意味著其他領域就不重要。事實上，多元化的專業領域對於一個國家的綜合發展是至關重要的。

**Comment by titan92601** [comment]
我認為台灣重視電機系因為半導體產業需求大、薪資高，加上社會對電機系的高科技刻板印象，使其更有吸引力。機械系相對資源較少，發展受限。然而，過度依賴單一產業存在風險，多元化發展和跨學科整合才能提高經濟韌性和競爭力。
